javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,95 @@
+// Voice assistant
+
+let name = "";
+let todo = [];
+function getReply(command) {
+  let phraseArr = command.toLowerCase().split(" ");
+
+  if (
+    phraseArr[phraseArr.indexOf("is")] &&
+    phraseArr[phraseArr.indexOf("is") + 1] &&
+    !phraseArr.includes("what")
+  ) {
+    name = phraseArr.slice(phraseArr.indexOf("is") + 1, phraseArr.length);
+    console.log("nice to meet you " + name.join(" "));
```

Comment:
Its good to try with the string properties, but you can make it much simpler here. There is no need to slice or join
For example check whether the phraseArr contains 'name' and '!what' . If true you can simple console it `nice to meet you ${phraseArr[phraseArr.length-1]}`

Reply:
Thank you so much for reviewing and the advice's , going to work on improve my coding